docs/rpc-endpoints.md

```diff
@@ -216,6 +216,19 @@ Where data is the hex serialization of the variable value.
 This endpoint also accepts a querystring parameter `?proof=` which when supplied `0`, will return the
 JSON object _without_ the `proof` field.
 
+### GET /v2/constant_val/[Stacks Address]/[Contract Name]/[Const Name]
+Attempt to fetch a constant from a contract. The contract is identified with [Stacks Address] and 
+ [Contract Name] in the URL path. The constant is identified with [Const Name].
+
+Returns JSON data in the form:
+```
+{
+  "data": "0x01ce...",
```

Comment:
Could we return the type and perhaps a deserialized representation of the value?

Reply:
This would be inconsistent with all the other endpoints behaviour.
I don't think we should do that.